docs/ipv6-support.md

```diff
@@ -0,0 +1,42 @@
+## Overview
+This document explains IPv6 support in AKO.
+
+### Use cases
+AKO supports IPv6 and dual stack. Following table explains what all is supported in AKO.
+| Use case   |      Dual stack support      |  comments |
+|----------|---------------|-------|
+| Frontend VIP |  SUPPORTED | Customers can choose to have either v6only or dual VIPs (v4, v6) for the Virtual services |
+| Backend (Pod IPs) |    V6ONLY   |   AKO will add either v4 or v6 addresses to the pools based on AKO configuration ('ipFamily'). AKO does not support a mix of v4 and v6 addresses |
+| K8s Nodes | V6ONLY |    AKO configures routes to the pod IPs via Node IP. AKO will choose v4 or a v6 IP based on ipFamily attribute in AKO config. For NodePort mode, AKO will choose either v4 or v6 IPs of Nodes based on the ipFamily attribute in AKO config |
+| Avi Controller IP | V4ONLY | AKO only supports v4 IPs to talk to Avi controller mgmt interface. AKO pod needs IPv4 connectivity to Avi controller mgmt interface |
+| K8s API server | V4ONLY | AKO only supports IPv4 to talk to K8s API server. v6 is not supported |
+
+### Support Matrix
+Cloud: vCenter
```

Comment:
please add this in a table for better readability.

Reply:
updated